content/sigs/docs/index.adoc

```diff
@@ -0,0 +1,123 @@
+---
+layout: sig
+title: "Documentation"
+section: sigs
+sigId: "docs"
+logo: /images/logos/formal/256.png
+tags:
+  - docs
+  - docs_sig
+  - docs-sig
+  - documentation
+leads:
+- name: "Mark Waite"
+  id: "markewaite"
+  github: "markewaite"
+  irc: "markewaite"
+  twitter: "MarkEWaite"
+participants:
+- name: "Oleg Nenashev"
+  id: "oleg_nenashev"
+  github: "oleg-nenashev"
+  irc: "oleg_nenashev"
+  twitter: "oleg_nenashev"
+- name: "Tracy Miranda"
+  id: "tracymiranda"
+  github: "tracymiranda"
+  irc: "tracymiranda"
+  twitter: "tracymiranda"
+- name: "Tammy Fox"
+  id: "tfoxnc"
+  github: "tfoxnc"
+- name: "Kristin Whetstone"
+  id: "kwhetstone"
+  github: "kwhetstone"
+  twitter: "lighteningdrake"
+- name: "Radek Antoniuk"
+  id: "warden"
+  github: "warden"
+  irc: "warden"
+- name: "John Ha"
+  id: "jha-cloudbees"
+  github: "jha-cloudbees"
+- name: "Petra Sargent"
+  id: "petra-sargent"
+  github: "petra-sargent"
+  twitter: "momsthatcode"
+- name: "Owen Mehegan"
+  id: "omehegan"
+  github: "omehegan"
+  twitter: "literatesavant"
+- name: "Meg McRoberts"
+  id: "stackscribe"
+  github: "stackscribe"
+- name: "Zhao Xiaojie (Rick)"
+  id: "surenpi"
+  github: "LinuxSuRen"
+links:
+  gitter: "jenkinsci/docs"
+  googlegroup: "jenkinsci-docs"
+  meetings: "https://docs.google.com/document/d/1uNNo0QJKPHnNp8PGr_jLI8p3K_94ZYD-M0evZOEZ93c/edit#"
+overview: >
+  This special interest group improves Jenkins use and adoption through documentation.
+  The SIG encourages, creates, and reviews documentation contributions and improves documentation with contributors and external communities.
+---
+
+The Documentation special interest group offers a venue for all kinds of documentation creation, improvement, and delivery.
```

Comment:
Interesting that, in this one sentence, you spell out "special interest group", not even capitalized, whereas it's SIG in the rest of the document.